lib/main_common.pm

```diff
@@ -963,7 +963,7 @@ sub load_consoletests {
     }
     loadtest "console/vim" if is_opensuse || !sle_version_at_least('15') || !get_var('PATTERNS') || check_var_array('PATTERNS', 'enhanced_base');
     # textmode install comes without firewall by default atm on openSUSE
-    if ((is_sle || !check_var("DESKTOP", "textmode")) && !is_staging() && !is_krypton_argon) {
+    if ((is_sle || !check_var("DESKTOP", "textmode")) && !is_staging() && !is_krypton_argon && !is_virtualization_server) {
```

Comment:
Could you please reference the fate issue - there was one, right? - that describes why this is the case?

Reply:
Added to the description.

Reply:
Sorry, I mean add as a comment in the source code